airflow/providers/fab/auth_manager/security_manager/override.py

```diff
@@ -2571,11 +2571,12 @@ def _get_microsoft_jwks(self) -> list[dict[str, Any]]:
 
     def _decode_and_validate_azure_jwt(self, id_token: str) -> dict[str, str]:
         verify_signature = self.oauth_remotes["azure"].client_kwargs.get("verify_signature", False)
+        claims_options = self.oauth_remotes["azure"].client_kwargs.get("claims_options", None)
```

Comment:
I think we should have somewhere in the docs explanation how to authenticate with Azure (not sure if in the Fab provider or Azure?) but we need this parameter explained in that sections so users will be aware how to set and use it

Reply:
Agree I will add the documentation in the  provider documents